features/speak-as.yml

```
@@ -0,0 +1,14 @@
+name: speak-as
+description: The `speak-as` CSS property sets how any element's content is spoken.
+# Note: This feature is distinct from `speak-as` in a @counter-style rule.
```

Let's tell developers directly that we too can be confused by this. 😅 

```suggestion
description: The `speak-as` CSS property sets how any element's content is spoken. Not to be confused with the `speak-as` descriptor of `@counter-style` at-rules.
```